pilot/pkg/networking/util/util_test.go

```diff
@@ -649,3 +650,47 @@ func TestCustomHandleCrash(t *testing.T) {
 
 	panic("test")
 }
+
+func TestIsAllowAnyOutbound(t *testing.T) {
+	tests := []struct {
+		node   *model.Proxy
+		result bool
+	}{
+		{
+			node:   &model.Proxy{},
+			result: false,
+		},
+		{
+			node: &model.Proxy{
+				SidecarScope: &model.SidecarScope{},
+			},
+			result: false,
+		},
+		{
+			node: &model.Proxy{
+				SidecarScope: &model.SidecarScope{
+					OutboundTrafficPolicy: &networking.OutboundTrafficPolicy{
+						Mode: networking.OutboundTrafficPolicy_REGISTRY_ONLY,
+					},
+				},
+			},
+			result: false,
+		},
+		{
+			node: &model.Proxy{
+				SidecarScope: &model.SidecarScope{
+					OutboundTrafficPolicy: &networking.OutboundTrafficPolicy{
+						Mode: networking.OutboundTrafficPolicy_ALLOW_ANY,
+					},
+				},
+			},
+			result: true,
+		},
+	}
+	for i := range tests {
+		out := IsAllowAnyOutbound(tests[i].node)
+		if out != tests[i].result {
+			t.Errorf("Expected %t but got %t for test case: %v\n", tests[i].result, out, tests[i].node)
```

Comment:
nit: suggest `t.Run` to do subtests rather than multiple cases in one test

Reply:
done